packages/tracing-operations/open-api/api-tracing-operations-interop-v1.yaml

```
@@ -0,0 +1,611 @@
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/TracingStateUpdateRequest"
+      responses:
+        "204":
+          description: No content
+        "400":
+          description: The provided input data is invalid
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem400"
+        "404":
+          description: The requested resource could not be found on the server
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem404"
+        "500":
+          description: A managed error has occurred during the request elaboration
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem500"
+
+  /tracings/{tracingId}/versions/{version}/savePurposeError:
+    post:
+      summary: Save purpose error
+      operationId: savePurposeError
+      tags:
+        - Write
+      parameters:
+        - name: tracingId
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: version
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/SavePurposeErrorRequest"
+      responses:
+        "204":
+          description: No content
+        "400":
+          description: The provided input data is invalid
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem400"
+        "500":
+          description: A managed error has occurred during the request elaboration
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem500"
+
+  /tracings:
+    get:
+      summary: Get all tracings of a tenant
+      operationId: getTracings
+      tags:
+        - Read
+      parameters:
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetTracingsByTenantId"
+        "500":
+          description: A managed error has occurred during the request elaboration
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Problem500"
+
+  /tracings/deletePurposesError:
```

probably we can move the logic in the external service, and change this endpoint to
```suggestion
  /tracings/{tracingId}/versions/{version}/errors:
```